src/Microsoft.Identity.Web/MicrosoftGraph/MicrosoftGraphServiceExtensions.cs

```diff
@@ -0,0 +1,93 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using Microsoft.Graph;
+
+namespace Microsoft.Identity.Web
+{
+    /// <summary>
+    /// Extensions methods on a MicrososoftAppCallingWebApiAuthenticationBuilder builder
+    /// to add support to call Microsoft Graph.
+    /// </summary>
+    public static class MicrosoftGraphServiceExtensions
+    {
+        /// <summary>
+        /// Add support to calls Microsoft graph. From a named option and a configuration section.
+        /// </summary>
+        /// <param name="builder">Builder.</param>
+        /// <param name="configurationSection">Configuraiton section.</param>
```

Comment:
```suggestion
        /// <param name="configurationSection">Configuration section.</param>
```

Reply:
fixed in this [PR](https://github.com/AzureAD/microsoft-identity-web/pull/442)

---
In reply to: [469487409](https://github.com/AzureAD/microsoft-identity-web/pull/434#discussion_r469487409) [](ancestors = 469487409)